frontend/src/components/Profile/ProfileUserContent.js

```diff
@@ -0,0 +1,46 @@
+import { Card } from 'antd';
+import {useState} from "react";
+import ProfilePostsTab from "./ProfilePostsTab";
+import ProfileThreadsTab from "./ProfileThreadsTab";
+
+const tabList = [
+    {
+        key: 'threads',
+        tab: 'Threads',
+    },
+    {
+        key: 'posts',
+        tab: 'Posts',
+    },
+];
+
+const ProfileUserContent = ({username, posts, threads, displayName}) => {
+    const [activeTabKey, setActiveTabKey] = useState('tab1');
```

Comment:
I'm probably a bit confused but shouldn't we set this initially to either threads or posts?